ghostwriter-jdk-v7/src/main/java/io/ghostwriter/openjdk/v7/GhostWriterAnnotationProcessor.java

```diff
@@ -50,8 +50,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             }
         }
 
-        final boolean doClaimAnnotations = false;
-        return doClaimAnnotations;
+        return false;
```

Comment:
Making the code easy to understand without commenting is a challenge, mostly because of the Java API naming.
Most probably inserting 1 line of comment is the best solution here. As an alternative JavaDoc might of help. Is that not an option?

This is my best idea regarding a comment-free solution: 
http://pastebin.com/LENWcstW
What do you think?